Dockerfile

```diff
@@ -5,12 +5,18 @@ LABEL version="0.9.2"
 LABEL description="A simple HTTP service."
 LABEL org.kennethreitz.vendor="Kenneth Reitz"
 
-RUN apt update -y && apt install python3-pip -y
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
 
-EXPOSE 80
+RUN apt update -y && apt install python3-pip git -y && pip3 install --no-cache-dir pipenv
+
+ADD Pipfile Pipfile.lock /httpbin/
+WORKDIR /httpbin
+RUN /bin/bash -c "pip3 install --no-cache-dir -r <(pipenv lock -r)"
 
 ADD . /httpbin
+RUN pip3 install --no-cache-dir /httpbin
 
-RUN pip3 install --no-cache-dir gunicorn /httpbin
+EXPOSE 80
```

Comment:
I wonder why the was moved here? (Also the ‘WORKDIR’). Couldn’t we move them again to top so we can avoid adding additional  docker layers for these if ‘Pipfile.lock’ changes?